### PR TITLE
refactor: airtable tables into own file with fields schema as JSON

### PR DESCRIPTION
**Pull request by mjmaurer**
Moves table-specific airtable logic into separate table file.

Also introduces a schema as JSON that can be partially generated by running a script on the airtable API webpage.

### CODE REVIEW

File: src/lib/airtable/tables/__schemaGen.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+// You can enter this in the console on:
```
Comment (piratefsh) [inline src/lib/airtable/tables/__schemaGen.js:1]: im a little confused about that the inputs and outputs/sideeffects of this script is, can you elaborate?

Comment (mjmaurer) [reply src/lib/airtable/tables/__schemaGen.js:1]: For sure. I'll add some more docs and work it around a bit. Very goofy hack

Comment (mjmaurer) [reply src/lib/airtable/tables/__schemaGen.js]: done

**Comment by mjmaurer** [comment]
I'm gonna go ahead and merge. Alex if you have any issues I'll be happy to make a change

I think having a top level fields object is a good balance between flexability, readability and protecting us from field naming bugs

File: src/lib/airtable/tables/__tableSchemaGen.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+// Because this script isn't perfect and field names can be long,
+// you should go through and normalize field accessors manually.
+
+// Updates after the initial table creation should probably be
+// done manually. This could still be useful for grabbing long
+// select-field options as JSON.
+
+/* eslint-disable no-undef */
+tableNameToGenerate = "Requests";
+camelCase = s =>
```
Comment (alexquick) [inline src/lib/airtable/tables/__tableSchemaGen.js:24]: this is gross and I love it so much

**Pull request merged**